### DISCUSSION TITLE
How to restrict wiki access for whoever except my company

### DISCUSSION THREAD

**Discussion by PierreMoreno** [requarks/wiki#6757] [Help / Questions] [answered]
Hello, 

I just installed wiki.js few days ago, and I'm trying to block access to my wiki. I just want to open port at my company's ip.
I tried with iptables to open only 80 443 and it didn't work. So I tried to open 3000, 5432 & 3443 ( port redirection with docker) but it didn't work too. 

On other ip than my company we're block, like I want. But my company IP is block too.

What is the good process to do what I want ? 
Thanks in advance for helping me !

**Comment by eugene-rozov** [comment, marked as answer]
You can specify a proxy (like nginx) through which you access the wikijs installation. And on the proxy level you can whitelist only specific IPs for accessing the wiki.
